app/src/main/res/layout/hero_item.xml

```diff
@@ -1,58 +1,58 @@
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginLeft="@dimen/margin_default"
+    android:layout_marginTop="@dimen/margin_default"
+    android:layout_marginRight="@dimen/margin_default"
+    android:layout_marginBottom="@dimen/margin_default"
+    android:orientation="vertical"
+    tools:cardElevation="2dp"
+    tools:cardUseCompatPadding="true">
+
 
     <androidx.constraintlayout.widget.ConstraintLayout
-            android:layout_width="match_parent"
-            android:layout_height="match_parent">
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
 
         <ImageView
-                android:id="@+id/heroCardImage"
-                android:layout_width="@dimen/dp_0"
-                android:layout_height="@dimen/dp_200"
-                android:scaleType="centerCrop"
-                android:src="@drawable/spider_man_cover"
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintTop_toTopOf="parent"
-                app:layout_constraintStart_toStartOf="parent"
-        />
+            android:id="@+id/heroCardImage"
+            android:layout_width="@dimen/dp_0"
+            android:layout_height="@dimen/dp_200"
+            android:scaleType="centerCrop"
+            android:src="@drawable/spider_man_cover"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
 
         <TextView
-                android:id="@+id/heroName"
-                android:text="@string/hero_name"
-                android:maxLines="1"
-                android:textAppearance="@style/Base.TextAppearance.AppCompat.Large"
-                android:layout_width="@dimen/dp_0"
-                android:layout_height="wrap_content"
-                android:ellipsize="end"
-                android:padding="@dimen/padding"
-                android:layout_marginRight="@dimen/margin_default"
-                android:layout_marginEnd="@dimen/margin_default"
-                app:layout_constraintTop_toBottomOf="@+id/heroCardImage"
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintBottom_toBottomOf="parent"
-                app:layout_constraintEnd_toStartOf="@+id/favoriteIcon"
-        />
+            android:id="@+id/heroName"
+            android:layout_width="@dimen/dp_0"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/margin_default"
+            android:layout_marginRight="@dimen/margin_default"
+            android:ellipsize="end"
+            android:maxLines="1"
```

Comment:
Should be a grate idea create an integer file, to set this and other integer elements?

Reply:
i create integers file and forget maxLines and remains magic numbers.... in next commits i will fix it.

Reply:
done